.github/actions/systemtests/run-perf-report/generate-report.js

```diff
@@ -0,0 +1,524 @@
+const fs = require('fs');
```

Comment:
so you have some output from tests which is in txt format and you basically parse it and create md for report? Wouldn't be better to have as part of the tests already md file? I mean you have logic for report generator twice now, or?